pkg/controllers/secrets.go

```diff
@@ -206,3 +215,19 @@ func SelectSecrets(secrets map[string]string, secretsToSelect []string) (map[str
 
 	return secrets, nil
 }
+
+func CheckForDangerousSecretNames(secrets map[string]string) error {
+	dangerousSecretNamesFound := []string{}
+
+	for _, dangerousName := range dangerousSecretNames {
+		if _, ok := secrets[dangerousName]; ok {
+			dangerousSecretNamesFound = append(dangerousSecretNamesFound, dangerousName)
+		}
+	}
+
+	if len(dangerousSecretNamesFound) > 0 {
+		return fmt.Errorf("your config contains the following potentially dangerous secret names:\n- %s", strings.Join(dangerousSecretNamesFound, "\n- "))
```

Comment:
Mentioned elsewhere: I think it might be helpful to include a link to our docs page that talks about why these values are considered dangerous.

Reply:
I should have mentioned this, but I'm unsure where to put the docs for this. I'm thinking README but should it be under an existing section (such as `COMMAND LINE`) or somewhere else. 

The discussion is here: https://linear.app/doppler/issue/ENG-5001#comment-4e17b130

Reply:
Can you add the docs link to the error?

Reply:
I have it in two places currently:
https://github.com/DopplerHQ/cli/blob/c48ee51eceb87b2706be1fda11e1d51dfa06a4db/pkg/controllers/secrets.go#L35-L36
https://github.com/DopplerHQ/cli/blob/c48ee51eceb87b2706be1fda11e1d51dfa06a4db/pkg/controllers/secrets.go#L244-L246

Would you still like me to add it to this error?